packages/gatsby/src/bootstrap/index.js

```diff
@@ -385,7 +386,16 @@ module.exports = async (args: BootstrapArgs) => {
 
   const graphqlRunner = (query, context = {}) => {
     const schema = store.getState().schema
-    return graphql(schema, query, context, context, context)
+    return graphql(
+      schema,
+      query,
+      context,
+      {
+        ...context,
+        nodeModel: nodeStore,
```

Comment:
We should also think about if we want to provide something for getting nodes of Interface and Union types, i.e. those that don't map 1:1 to node types in the store?